content/academy/expert_scraping_with_apify/migrations_maintaining_state.md

```diff
@@ -0,0 +1,46 @@
+---
+title: VI - Migrations & maintaining state
+description: Learn about what actor migrations are and how to handle them properly so that state is not lost and it can safely be resurrected.
+menuWeight: 6.6
```

Comment:
This will need updating so Solutions will be at the end
<img width="286" alt="Screenshot 2022-05-04 at 09 14 48" src="https://user-images.githubusercontent.com/38246758/166644774-ead1cee2-690b-48cf-a949-4a1ac4abb1fb.png">